tests/queries/0_stateless/02211_shcema_inference_from_stdin.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+$CLICKHOUSE_LOCAL -q "select toUInt32(number) as x from numbers(10) format JSONEachRow" > data.jsoneachrow
```

Comment:
Wa can use `.jsonl` extension https://jsonlines.org/

```suggestion
$CLICKHOUSE_LOCAL -q "select toUInt32(number) as x from numbers(10) format JSONEachRow" > data.jsonl
```

Reply:
Ok, let's add support of this extension: https://github.com/ClickHouse/ClickHouse/pull/34496